lib/services/local/presence.go

```diff
@@ -30,12 +31,20 @@ import (
 // PresenceService records and reports the presence of all components
 // of the cluster - Nodes, Proxies and SSH nodes
 type PresenceService struct {
+	*log.Entry
 	backend.Backend
+	// fast getter of the items extension
```

Comment:
What about something like `getter is used to make batch requests to the backend.`